aries-site/src/pages/foundation/date-and-time.mdx

```diff
@@ -0,0 +1,283 @@
+  timeStyle: 'short',
+};
+
+| Locale | Date                                                            | Time                                                          |
+| ------ | --------------------------------------------------------------- | ------------------------------------------------------------- |
+| en-US  | {Intl.DateTimeFormat('en-US', localeFormat).format(localeDate)} | {Intl.DateTimeFormat('en-US', timeFormat).format(localeDate)} |
+| en-AU  | {Intl.DateTimeFormat('en-AU', localeFormat).format(localeDate)} | {Intl.DateTimeFormat('en-AU', timeFormat).format(localeDate)} |
+| de     | {Intl.DateTimeFormat('de', localeFormat).format(localeDate)}    | {Intl.DateTimeFormat('de', timeFormat).format(localeDate)}    |
+
+### Timezones
+
+In addition to formatting a date according to the user's locale, present time in the user's timezone and display the timezone alongside the time stamp for clarity.
+
+In instances when a user's timezone cannot be determined, output the date and time in UTC+0.
+
+<Notification
+  status="info"
+  message="The same moment in time will be presented differently based on the user's timezone and locale."
+  width={{ max: 'large' }}
+  margin={{ bottom: 'medium' }}
+/>
+
+<BestPracticeGroup>
+  <Example
+    height="none"
+    bestPractice={{
+      type: 'do',
+      message:
+        'Utilize the browser locale to format and update the date and time for users, providing a personalized experience.',
+    }}
+  >
+    <TimezoneExample />
+  </Example>
+  <Example
+    height="none"
+    bestPractice={{
+      type: 'dont',
+      message: `Don't present a date in a different timezone than that of the user. This leads to confusion, increased cognitive load, and sets users up for failure.`,
+    }}
+  >
+    <TimezoneExample bestPractice={false} />
+  </Example>
+</BestPracticeGroup>
+
+### ISO 8601 Formatting
+
+ISO 8601 is an internationally recognized standard for presenting date and time. It is presented as `YYYY-MM-DDTHH:mm:ss.sssZ`.
+
+For example, `2023-02-07T06:57:00.000Z`.
+
+In this format, `T` indiciates the start of the time stamp, and `Z` indiciates the time is in UTC.
+
+Some use cases include:
+
+- Log files
+- Trigger times
+
+Even though ISO 8601 is an internationalized standard, it is not the most friendly format for users. In most other cases, date and time formatting should be based on the user's [locale](#locale-formatting) and [timezone](#timezones).
+
+## Relative time
+
+Relative time stamps display:
+
+1. How recently an event has occurred.
+1. How far in the future an event will occur.
+
+Relative time stamps remove the need for the user to perform mental calculations by presenting timestamps in written form like "2 days ago".
+
+When displaying relative time:
+
+- Adapt and show the appropriate unit of time. For example, after 60 minutes, display hours.
+- Unless required, displaying the time for anything less than a second and more than a week is generally not needed.
```

Comment:
Another bullet point here stating that "Zero values should not be included as this will cause issues with alignment. Ex. do not show when something is at 0 minutes."

Reply:
Done.